src/Profile/PathValue.cpp

```diff
@@ -49,6 +51,40 @@ ProfileMap::GetPath(const char *key) const
   return ExpandLocalPath(Path(buffer));
 }
 
+std::vector<AllocatedPath> 
+ProfileMap::GetMultiplePaths(const char *key) const{
+
+	std::vector<AllocatedPath> paths;
+	TCHAR buffer[MAX_PATH];
+
+	if(!Get(key, buffer, ARRAY_SIZE(buffer)))
+		return paths;
+
+	if (StringIsEmpty(buffer))
+		return paths;
+
+	StringView string_view(buffer);
+
+	IterableSplitString path_splitstring(string_view, ':');
+
+	for (auto i = path_splitstring.begin(); i != path_splitstring.end(); ++i){
+
+		if ((*i).empty())
```

Comment:
Even if an iterator were a good idea here: just do `i->empty()` instead of `(*i).empty()`. Don't make it more complicated that it needs to be!